src/openrct2/ride/RideData.h

```diff
@@ -274,6 +274,8 @@ enum ride_type_flags : uint64_t
     RIDE_TYPE_FLAG_SUPPORTS_LEVEL_CROSSINGS = (1ULL << 49),
     RIDE_TYPE_FLAG_IS_SUSPENDED = (1ULL << 50),
     RIDE_TYPE_FLAG_HAS_LANDSCAPE_DOORS = (1ULL << 51),
+    RIDE_TYPE_FLAG_GENTLE_INCLINE_ONLY = (1ULL << 52),
```

Comment:
```suggestion
    RIDE_TYPE_FLAG_GENTLE_UP_INCLINE_ONLY = (1ULL << 52),
```

This hopefully makes it clearer that steep downward inclines are still allowed.

Reply:
I've renamed the flags